src/sql/parts/notebook/notebook.component.ts

```diff
@@ -135,7 +134,7 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		}
 	}
 
-	public get model(): NotebookModel {
+	public get model(): NotebookModel | null {
```

Comment:
Out of curiosity, when are we trying to get the model before it's been created?

Reply:
Anytime before modelReady is resolved it's _possible_. Not sure if we're really doing it, this is to match the overall API.